build.gradle

```diff
@@ -12,21 +12,18 @@ plugins {
 group = 'com.tokbox'
 archivesBaseName = 'opentok-server-sdk'
 version = '4.14.0'
-sourceCompatibility = "1.8"
-targetCompatibility = "1.8"
 
-ext.githubPath = 'opentok/opentok-java-sdk'
+ext.githubPath = "opentok/$archivesBaseName"
```

Comment:
I think the change of this line is wrong, see https://github.com/opentok/Opentok-Java-SDK/pull/274

Reply:
Good spot!